terminal/theme.py

```
@@ -0,0 +1,77 @@
+    "test": Theme(
+        name="Test Theme",
+        prompt_style="bold bright red",
+        response_style="bold bright green", 
+        error_style="bold bright yellow",
+        success_style="bold bright blue",
+        info_style="bold bright cyan",
+        warning_style="bold bright magenta",
+    ),
+}
+
+def get_theme(name: str = None) -> Theme:
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)